docs/topics/migrations.txt

```diff
@@ -140,6 +140,13 @@ developers (or your production servers) check out the code, they'll
 get both the changes to your models and the accompanying migration at the
 same time.
 
+.. versionadded:: 1.8
```

Comment:
Yes. I was about to backport the patch to 1.8 as the current behavior is more fragile than helpful.